tests/types/test_enum8.py

```diff
@@ -9,6 +9,8 @@
 
 
 class TestEnum(enum.IntEnum):
+    __test__ = False
```

Comment:
Without:
```shell
.../clickhouse-sqlalchemy/tests/types/test_enum8.py:11: PytestCollectionWarning: cannot collect test class 'TestEnum' because it has a __new__ constructor (from: tests/types/test_enum8.py)
    class TestEnum(enum.IntEnum):
```